tests/publiccloud/img_proof.pm

```diff
@@ -92,6 +93,18 @@ sub run {
     # Because the IP address of instance might change during img_proof due to the hard-reboot, we need to re-add the ssh public keys
     assert_script_run(sprintf('ssh-keyscan %s >> ~/.ssh/known_hosts', $instance->public_ip));
 
+    if (is_hardened) {
+        # Add soft-failure for https://bugzilla.suse.com/show_bug.cgi?id=1220269
+        zypper_call "in jq";
+        my $outcome = script_output "cat $img_proof->{results} | jq '.tests[] | select(.nodeid | startswith(\"test_sles_hardened\")) | .outcome'";
```

Comment:
This should work, but have you considered parsing the json within the test code itself to avoid installing `jq` on the instance itself? :thinking: 

This is just a thought, not a requirement.

Reply:
> This should work, but have you considered parsing the json within the test code itself to avoid installing `jq` on the instance itself? 🤔
> 
> This is just a thought, not a requirement.

We would have to load the JSON, modify it, then write it again to a file se we can upload it later in `parse_extra_log(IPA => $img_proof->{results});`

 I will research the best way to do it in Perl.

Reply:
There is a very long JSON programming, making difficult to verify the real output obtained.
If possible, I'd kindly suggest, if not already done, some aside unit test of that only json code.

Only for clarification, being Not so clear to me what its scope, but, is there no other possible way to achieve that requirement?

Reply:
I will do it in Perl.